stories/Organism/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react';
+import { FaqFun, SidebarMenu } from '../../assets/js/Faq_custom'
+import './sidebar.scss';
+
+export const Sidebar = ({ headerText, data, label }) => {
+  useEffect(() => {
+    FaqFun();
+    SidebarMenu();
+  }, [])
+
+  return (
+    <div className='grid-x grid-margin-x'>
+      <div className='cell medium-4 accordion-wrapper'>
+        <li className='accordion-menu' id='sidebar-menu'><span>{label}</span></li>
```

Comment:
@pallavikadam-srijan can you use li outside of ul and it's class should be accordion__menu as per BEM